### ISSUE TITLE
IPv6 not disabling on Ubuntu 20.04 with netplan

### ISSUE THREAD

**Issue by stickz** [pia-foss/manual-connections#70]
I noticed traffic was routing outside my pia wireguard interface through IPv6. run_setup.sh was told to disable IPv6. It ran the commands to do so, however, these commands were not effective in disabling IPv6. 

```
sysctl -w net.ipv6.conf.all.disable_ipv6=1
sysctl -w net.ipv6.conf.default.disable_ipv6=1
```

I added the option to my grub boot loader to disable it, but that didn't work ether. It turns out that netplan was keeping it enabled. It finally got disabled when I removed the ipv6 gateway & IP address from the netplan configuration file.

Long story short, anther solution is required to disable IPv6. This method may not work on newer Ubuntu systems.

**Comment by g00nix** [comment]
`sysctl` manages the Linux kernel parameters at runtime. Those commands will work on any Linux distribution, and the documentation can be found on the official kernel website: https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt

If you did find such a big bug in a very popular distribution, please open this bug within the official bug trackers for the kernel and for Ubuntu:
- https://bugzilla.kernel.org/
- https://bugs.launchpad.net/ubuntu
